pom.xml

```diff
@@ -150,10 +150,9 @@
     <parquet.version>1.11.0</parquet.version>
     <protobuf.version>3.12.4</protobuf.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <ratis.version>2.0.0</ratis.version>
+    <ratis.version>2.2.0</ratis.version>
```

Comment:
@jenoudet fysa. if you are wary of this version upgrade, we could first upgrade the ratis version in a separate PR and check if there are any issues with it before merging the rest of the ozone related changes

Reply:
Indeed, we don't need to bump ratis version to `2.2.0` as it brings backward compatible issue. Revert the related change, thanks for the remind.